server/tests/controllers/users.js

```diff
@@ -46,6 +46,8 @@ const invalidPassword = {
   email: 'emeka1@gmail.com',
   password: 'sdfghgfdkjshfbjbsfjbdmnbdkjfgeugfuefguefge'
 };
+const worngEmail = { email: 'xyz@mail.com', password: 'passtes' };
```

Comment:
There is a typo here, you have ```worngEmail```, you might want to change it to ```wrongEmail``` to aid easy interpretation of your code

Reply:
Thanks bro. Will correct this